docs/docs/user-setup-cli.mdx

```diff
@@ -0,0 +1,278 @@
+---
+title: (EXPERIMENTAL) Installing ZMK
+sidebar_label: (EXPERIMENTAL) Installing ZMK
+---
+
+This is an **experimental** command line program to help set up ZMK. [Click here to return to the main documentation](user-setup.mdx).
+
+ZMK CLI walks you through installing ZMK and setting up a GitHub repository to store and build custom firmware. It also automates some common tasks such as adding new keyboards to your repository.
+
+The instructions below contain commands that need to be run in a terminal program. On Windows, use the [Windows Terminal](https://apps.microsoft.com/detail/9n0dx20hk701) or PowerShell. On other operating systems, the terminal program is usually just named "Terminal".
+
+:::info
+If you use this program, any bug reports are highly appreciated. We are happy to help you fix your setup if you encounter a bug through the usage of this program.
+:::
+
+## Installation
+
+### Preliminaries
+
+#### Make sure you have Git installed
+
+```sh
+git --version
+```
+
+<details>
+  <summary>I don't have Git installed.</summary>
+
+Install Git from https://git-scm.com/downloads.
+
+If you have Windows 11, you can instead open a terminal and run:
+
+```sh
+winget install git.git
+```
+
+Re-open your terminal and test to make sure it is now installed.
+
+</details>
+
+#### Make sure you have Python installed
+
+```sh
+python --version
+python3 --version
```

Comment:
Depending on how your system is configured, only one of these commands may work. `python3` is most likely to work, so I recommend we start with that command. Then we can optionally say to try `python` if that didn't work.